components/TextFields/src/ContainedInputView/private/MDCTextControlStyleBase.m

```diff
@@ -20,10 +20,14 @@
 #import "MDCTextControl.h"
 #import "MDCTextControlVerticalPositioningReferenceBase.h"
 
+static const CGFloat kBaseFloatingLabelScaleFactor = 0.8;
```

Comment:
Is this customizable by clients?

Reply:
Nope. Maybe someday clients will want to specify it but I think we should cross that bridge when we get there